packages/twenty-front/src/modules/favorites/hooks/useFavoriteFolders.ts

```diff
@@ -0,0 +1,96 @@
+import { calculateNewPosition } from '@/favorites/utils/calculateNewPosition';
+import { CoreObjectNameSingular } from '@/object-metadata/types/CoreObjectNameSingular';
+import { useCreateOneRecord } from '@/object-record/hooks/useCreateOneRecord';
+import { useDeleteOneRecord } from '@/object-record/hooks/useDeleteOneRecord';
+import { useUpdateOneRecord } from '@/object-record/hooks/useUpdateOneRecord';
+import { OnDragEndResponder } from '@hello-pangea/dnd';
+import { usePrefetchedFavoritesData } from './usePrefetchedFavoritesData';
+
+export const useFavoriteFolders = () => {
```

Comment:
I think this should be split into multiple hooks, let's make sure to separate useFavoriteFoldersForNavigationDrawer from useFavoriteFolderPicker (dropdown)